src/auxinfo/info.rs

```diff
@@ -20,7 +20,7 @@ use serde::{Deserialize, Serialize};
 use tracing::error;
 
 /// The private key corresponding to a given Participant's [`AuxInfoPublic`].
-#[derive(Serialize, Deserialize, ZeroizeOnDrop)]
+#[derive(Clone, Serialize, Deserialize, ZeroizeOnDrop)]
```

Comment:
Yeah, I'm not sure the best approach here. We could implement a `LocalStorage::remove` method that avoids the need for a `Clone`, but we heavily rely on checking storage for values to see when we're done with the protocol (see e.g. `is_auxinfo_done`). _However_, I think we could just check for `PublicStorageType` since they are stored at the same time, so if one exists the other should too.

Unfortunately, we also need to move the value from `LocalStorage` to persistent storage; however, this may go away once we remove persistent storage, although getting that to work in this interim state might be tricky.

Reply:
I also realized that even without the `Clone`, main storage is serializing and deserializing, which has essentially the same affect as cloning.

Reply:
Yeah, that's fair. I don't think we should do anything about this right now, but I definitely want to revisit our zeroize approach later to become more confident that data is getting cleared.

Reply:
I added a TODO addressing #169 (which I think is the most relevant issue related to this)